pennylane/vqe/vqe.py

```diff
@@ -470,21 +471,23 @@ def __init__(
         """QNodeCollection: The QNodes to be evaluated. Each QNode corresponds to the expectation
         value of each observable term after applying the circuit ansatz."""
 
-        wires = device.wires.tolist()
-
         tape_mode = qml.tape_mode_active()
         if tape_mode:
+
+            d = device[0] if isinstance(device, Sequence) else device
+            w = d.wires.tolist()
```

Comment:
So if the user _does_ pass a list of devices, we will just silently use the first one here?

Should we raise a warning?

Reply:
(Still getting up to speed with recent changes, an extended question:) Could the VQE tutorial - which passes multiple devices - still be compatible with tape mode if we'd only use a single device?

Reply:
Correct! Yeah I did wonder if it was a bit implicit. But then I decided that since this is what happens in non-tape mode and no warning is raised, the precedent had been set :laughing:

Reply:
Ok, I added a warning for both!